README.md

```diff
@@ -51,6 +51,11 @@ You can use a Web server and a supported database (MariaDB, MySQL or PostgreSQL)
  
 - Create an empty `htdocs/conf/conf.php` file and set *write* permissions for your web server user (*write* permission will be removed once install is finished)
 
+- Run composer 
+```console 
+composer update
```

Comment:
If you run a composer update here, you will break a lof ot thing in dolibarr.
For example, the ckeditor provided with composer does not works with dolibarr when official package has no trouble. TCPDF needs also a patch that are lost after the update.
EScpos is broken too. Restler too because patch fixes are also lost, etc...
It is better to not recommend to use composer manually and get the update of libraries by updating dolibarr with maintenance releases.

Reply:
I see, vendor directory is commited in the project, is not a normal behaivor. I strongly recomend make a pull request in original project for made the correct patchs... For example you can extends or override a behaivor of an external library but without touch the original code. The purpose of using a dependency manager is to be able to update them easily.